LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/MessageDispatcher.cs

```diff
@@ -0,0 +1,315 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoRaWan.NetworkServer
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text;
+    using System.Threading.Tasks;
+    using LoRaTools;
+    using LoRaTools.LoRaMessage;
+    using LoRaTools.LoRaPhysical;
+    using LoRaTools.Regions;
+    using LoRaTools.Utils;
+    using Microsoft.Azure.Devices.Client;
+    using Microsoft.Extensions.Logging;
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// Message dispatcher
+    /// </summary>
+    public class MessageDispatcher
+    {
+        // Defines Cloud to device message property containing fport value
+        internal const string FPORT_MSG_PROPERTY_KEY = "fport";
```

Comment:
arn't those the same as in Constants.cs?

Reply:
True, good catch